web/middleware/bbb.ts

```diff
@@ -0,0 +1,133 @@
+import { hashToken } from "../../common/util/hashing";
```

Comment:
This shouldn't be in the middleware folder, since it is no express middleware.

Reply:
Moved it to the utils folder for now.